### DISCUSSION TITLE
Pergunta

### DISCUSSION THREAD

**Discussion by nutritionsabino** [nutritionsabino/portfolio#2] [Q&A] [answered]
Como mudar o ponteiro do mouse pelo css? e também gostaria que ao clicar em uma imagem a mesma não fosse arrastada na tela

**Comment by marcosprofile** [comment, marked as answer]
Para mudar o ponteiro do mouse no css:
cursor: pointer; // ou você pode definir qual cursor gostaria de utilizar

Para impedir que uma imagem seja arrastada ao clicar e arrastar você pode incluir dentro da tag img a opção: draggable="false"
